app/frontend/src/components/FormList.vue

```diff
@@ -50,20 +50,20 @@ export default {
   data: () => ({
     // TODO: Consider loading part of this from API?
     forms: [
-      {
-        linkAdminName: 'AgriSeafoodOpScreeningAdmin',
-        linkName: 'AgriSeafoodOpScreeningForm',
-        resource: AppClients.AGRISEAFOODOPSCREENING,
-        roles: [AppRoles.VIEWER],
-        title: 'Agriculture and Seafood Operator Screening'
-      },
-      {
-        linkAdminName: 'ForestrySectorOpScreeningAdmin',
-        linkName: 'ForestrySectorOpScreeningForm',
-        resource: AppClients.FORESTRYSECTOROPSCREENING,
-        roles: [AppRoles.VIEWER],
-        title: 'Forestry Sector Operator Screening'
-      },
+      // {
+      //   linkAdminName: 'AgriSeafoodOpScreeningAdmin',
+      //   linkName: 'AgriSeafoodOpScreeningForm',
+      //   resource: AppClients.AGRISEAFOODOPSCREENING,
+      //   roles: [AppRoles.VIEWER],
+      //   title: 'Agriculture and Seafood Operator Screening'
+      // },
+      // {
+      //   linkAdminName: 'ForestrySectorOpScreeningAdmin',
+      //   linkName: 'ForestrySectorOpScreeningForm',
+      //   resource: AppClients.FORESTRYSECTOROPSCREENING,
+      //   roles: [AppRoles.VIEWER],
+      //   title: 'Forestry Sector Operator Screening'
+      // },
```

Comment:
Think we should just add an enabled boolean to the data object instead? If we need to keep flipping this for future feature toggling might be good to just have it be built in as logic instead of commenting it out.

Reply:
👍